recipes/yasm/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: yasm
```

Comment:
It could be, but it is not used in this case. Looks like they have code/binaries for Windows ( http://yasm.tortall.net/Download.html ) Will disable for now as we don't need it there.